content/en/docs/ops/diagnostic-tools/istioctl/index.md

```diff
@@ -101,23 +101,40 @@ See [Debugging Envoy and Istiod](/docs/ops/diagnostic-tools/proxy-cmd/) for more
 
 {{< tab name="macOS" category-value="macos" >}}
 
-If you are using the macOS operating system with the Bash terminal shell, make sure that the `bash-completion` package is installed. With the [brew](https://brew.sh) package manager for macOS, you can check to see if the `bash-completion` package is installed with the following command:
+If you are using the macOS operating system with the ZSH terminal shell, make sure that the `zsh-completions` package is installed. With the [brew](https://brew.sh) package manager for macOS, you can check to see if the `zsh-completions` package is installed with the following command:
```

Comment:
It's sort of hard to know if a brew package is installed using `brew info`. The `Not Installed` can be hard to find. Also, the single line of output here, is not really indicative of the real output.

Maybe
```
brew list zsh-completions                                                                                                                                                        
/usr/local/Cellar/zsh-completions/0.34.0/share/zsh-completions/ (147 files)
```
would be better to show installed. 
The command would yield a 
```
Error: No such keg: /usr/local/Cellar/zsh-completion
```
if it wasn't.

Reply:
Thanks, I shall make these changes

Reply:
Changes made, please do let me know if there are any other updates to make, thank you 😄